portfolio/src/main/webapp/style.css

```diff
@@ -45,6 +25,46 @@ body {
   max-width: 100%;
 }
 
+#navigation {
+  font-size: 5vw;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  color: white;
```

Comment:
[Alphabetize declarations.](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order)

Reply:
Updated with commit 436779b

Reply:
Updated in commit 436779b